test/MockSmartSyncPlugin.js

```diff
@@ -250,18 +250,19 @@ var MockSmartSyncPlugin = (function(window) {
                         self.sendUpdate(syncId, "FAILED", 0);
                     }
                 });
-            });        
+            });
         }
     };
 
     // Return module
     return module;
 })(window);
 
-var mockSyncManager = new MockSmartSyncPlugin(false);
-var mockGlobalSyncManager = new MockSmartSyncPlugin(true);
+
+var mockSyncManager = new MockSmartSyncPlugin({'isGlobalStore': false});
+var mockGlobalSyncManager = new MockSmartSyncPlugin({'isGlobalStore': true});
```

Comment:
Shouldn't you have a SyncManagerMap similar to the StoreMap you had in MockSmartStore?